pages/partials/examQuestionStatus.ejs

```diff
@@ -0,0 +1,20 @@
+<span class="align-middle">
+  <%
+  const badge_color = {
+    unanswered: 'danger',
+    invalid: 'danger',
+    grading: 'default',
+    saved: 'info',
+    complete: 'success',
+    correct: 'success',
+    incorrect: 'danger',
+  };
+  %>
+  <span class="badge badge-<%= badge_color[instance_question.status] ?? 'default' %>"><%= instance_question.status %></span>
```

Comment:
One aspect that may benefit from a review here is the distinction between incorrect, correct and complete in these cases. Given these are not often used in other contexts, they have been loosely updated. In essence:

* For exams: complete is a question that is correct and has no manual points, or with exhausted number of attempts. Correct is used for a question that has 100% auto-points but still has manual points, while incorrect is used for incorrect answers with attempts left.
* For homeworks: complete is not used. Correct is used if at least one submission got 100% auto-grading score. Incorrect is used otherwise.
* Manual grading right now always updates the status to complete.

We may want to revisit how these statuses are updated, or group these three into a different status, at least for student display. Other than this case, there is no other code that I could find (though I'm still looking a bit more thoroughly where this distinction is used, though there are cases where the other status values are important.

Reply:
After meeting with Matt, here's the decision:
* for homeworks, complete designates a question where max points have been achieved, or where manual grading has been done.
* everything else remains as is.

Reply:
Done in 49051d632.